### DISCUSSION TITLE
problems since updating to the new macOS Ventura

### DISCUSSION THREAD

**Discussion by sbuttigieg** [golangci/golangci-lint#3327] [Q&A] [answered]
Since updating to the latest macOS, I started getting errors similar to this one.

`can't run linter goanalysis_metalinter: gofmt: can't extract issues from gofmt diff output "--- /Users/myname.local/code/folder1/mocks/service.go.orig\t2022-10-26 11:05:33.....................": can't parse patch: parsing time "2022-10-26 11:05:33" as "2006-01-02 15:04:05 -0700": cannot parse "" as "-0700"`

Currently running v1.50.1

**Comment by ldez** [comment, marked as answer]
https://github.com/golangci/golangci-lint/issues/3087#issuecomment-1226508818

**Reply by sbuttigieg** [reply]
Thanks.
Don't know why it happened since the new macOS update, but an answer from last August saved my day.

**Reply by OscarVanL** [reply]
This also happened to me after updating to Ventura. So strange.

**Reply by slice-mrityunjoy** [reply]
same issue happened here as well

**Reply by dsonatus** [reply]
https://github.com/golangci/golangci-lint/issues/3087#issuecomment-1226508818
